python/triton/testing.py

```diff
@@ -120,10 +120,7 @@ def do_bench(fn, warmup=25, rep=100, grad_to_none=None, quantiles=None, fast_flu
     # before each kernel call to make sure that the L2 cache
     # doesn't contain any input data before the run
```

Comment:
Hi @int3! 

Just wondering why the buffer stopped clearing completely?

> to make sure that the L2 cache doesn't contain any input data before the run

Doesn't the current cleaning process contradict this? Or is the cleaning designed for GPUs with a cache less than 64 MB?

Reply:
I don't understand the question, or the suggested code changes -- isn't what you're proposing going to have exactly the same behavior?

The current code is designed to clear a 256MB cache

Reply:
My apologies, I read the code incorrectly.